.travis.yml

```
@@ -66,6 +66,7 @@ script:
   - cd ${TRAVIS_BUILD_DIR}/html
   - ../vendor/bin/drush -y si --db-url=mysql://drupal:drupal@127.0.0.1/drupal minimal
   - ../vendor/bin/drush -y cset system.site uuid $(grep uuid ${TRAVIS_BUILD_DIR}/config/system.site.yml | awk '{print $2}')
+  - ../vendor/bin/drush -y sqlc < ${TRAVIS_BUILD_DIR}/html/themes/contrib/common_design/_tests/sample.sql
```

Yesh! The sample.sql file is 16MB though, which is a bit crazy. Can you make sure you create that file by passing `-structure-tables-list="cache*,watchdog"` to `drush sql-dump` and then gzip the result as well, if it's over a few MB? If you do gzip it, the drush command to import it should change to:

```suggestion
  - zcat ${TRAVIS_BUILD_DIR}/html/themes/contrib/common_design/_tests/sample.sql.gz | ../vendor/bin/drush -y sqlc 
```